Tests/Fluent.Tests/Cdn/CdnTests.cs

```diff
@@ -17,7 +17,7 @@ public partial class Cdn
         private const string cdnEndpointName = "endpoint-f3757d2a3e10";
         private const string endpointOriginHostname = "mylinuxapp.azurewebsites.net";
 
-        [Fact(Skip = "Fails with server side issue, need to follow up with the CDN service team")]
+        [Fact(Skip = "test failed, need to fix later")]
```

Comment:
Please link to issue number.

Reply:
I will open issue for cdn test. For now it's like we don't have working test for cdn both Java and .NET side.